com.gitlab.coringao.JAG.yml

```
@@ -0,0 +1,35 @@
+app-id: com.gitlab.coringao.JAG
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: jag
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

Doesn't launch on Wayland.